gensim/summarization/summarizer.py

```diff
@@ -158,6 +158,11 @@ def summarize_corpus(corpus, ratio=0.2):
     _set_graph_edge_weights(graph)
     _remove_unreachable_nodes(graph)
 
+    # Cannot calculate eigenvectors if number of unique words in text < 3. Warns user to add more text. The function ends.
+    if len(graph.nodes()) < 3:
+        logger.warning("Please add more sentences to the text. The number of reachable nodes is below 3")
+        return
```

Comment:
This doesn't match the function signature (`Returns a list`). Either raise an exception, or return an empty list -- this is just nasty unexpected gotcha for users.